invite/test/e2e.test.ts

```diff
@@ -113,22 +122,20 @@ describe('end-to-end', () => {
             expect(body).toEqual({
               body: 'An invitation to join `test_org` has been sent to ' +
```

Comment:
Supernit: you don't always have to break long strings into multiple lines, especially if they're simple strings (i.e., not formatter strings) - this whole thing can be in one line and still pass Prettier formatting. Totally your choice of style

Reply:
I often have 3~4 vertical code panes, so I like keeping lines as narrow as the style guide will justify :P